README.md

```diff
@@ -85,21 +85,12 @@ You can also create an Azure service principal either through
     git clone https://github.com/Azure-Samples/azure-sdk-for-python-keyvault-secrets-get-set-managedid.git
```

Comment:
To make it clear that the following commands need to be run from inside the directory:
```suggestion
    git clone https://github.com/Azure-Samples/azure-sdk-for-python-keyvault-secrets-get-set-managedid.git
    cd azure-sdk-for-python-keyvault-secrets-get-set-managedid
```